packages/generators/app/src/utils/merge-template.ts

```diff
@@ -38,11 +39,16 @@ export default async function mergeTemplate(scope: Scope, rootPath: string) {
     scope.template?.startsWith(filePrefix)
   );
 
+  // Force templatePath to point to the internal example template
+  if (scope.useExampleApp) {
+    templatePath = path.resolve(__dirname, '..', 'resources', 'templates', 'example-app', scope.useTypescript ? 'ts' : 'js');
```

Comment:
You don't have the ts version yet from what I can see.

Reply:
Yes, right now the PR is not over, I'm just looking for feedback in terms of the way I implemented the system, not the template in itself :)

Reply:
Yes, not much to say it's ok considering we will move everything somewhere else soonish